CSharp/Common/SampleHelpers.cs

```diff
@@ -80,15 +81,15 @@ public static string ConstructBlobSource(string containerSasUrl, string blobName
 
             if (index != -1)
             {
-                //SAS                
+                //SAS
                 string containerAbsoluteUrl = containerSasUrl.Substring(0, index);
                 return containerAbsoluteUrl + "/" + blobName + containerSasUrl.Substring(index);
             }
             else
             {
                 return containerSasUrl + "/" + blobName;
             }
-        }
+         }
```

Comment:
Extra space?

Reply:
Not sure how I managed to do that, but good catch - fixed.